beyond/templates/cycles.html

```diff
@@ -0,0 +1,20 @@
+{% extends "layout.html" %}
```

Comment:
Not sure this is the information we want to place in the cycles page.

Reply:
So what info should we place there?

Reply:
The information on the past and current cycles :-)

Reply:
Cool, do you wish to handle it over to me, so i can stick it in the html?

Reply:
cycle, institute, school, semester, year
01, MTA, Information Systems, Winter 2020
02, MTA, Computer Science, Spring, 2020
03, IDC, Computer Science, Summer 2020
04, MTA, Information systems, Winter 2021
05, MTA, Computer Science, Spring 2021